src/covid_shared/paths.py

```diff
@@ -72,6 +73,7 @@
 GLOBAL_SYMPTOM_SURVEY_OUTPUT_DIR_NAME = Path('global')
 PULSE_SURVEY_OUTPUT_DIR_NAME = Path('pulse_survey')
 PULSE_SURVEY_US_CENSUS_OUTPUT_DIR_NAME = Path('US Census_pulse surveys')
+AUTOPROCESSING_CFR_AGE_OUTPUT_DIR_NAME = Path('auto_processing/CFR-age')
```

Comment:
Any concerns with this being a nested directory?

I assume it will all work but this looks like the only case of that.

Reply:
Good point - that's an easy enough fix. Just changed.

Reply:
If you _did_ want nested directories you can see how I did it w/ the pulse surveys. I had done that to keep it matching the way the data intake team had been saving them to OneDrive.